assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -123,14 +139,17 @@ export class CampaignSidebar extends Component {
         </PluginSidebarMoreMenuItem>
         <PluginSidebar
           name={ CampaignSidebar.getId() }
-          title={ __( 'Campaign Options', 'planet4-blocks-backend' ) }
+          title={ __('Campaign Options', 'planet4-blocks-backend') }
         >
-          { parent ? <PostParentLink parent={ parent }/> :
-            <LegacyThemeSettings
-              theme={theme}
-              handleThemeSwitch={ this.handleThemeSwitch }
-            />
-          }
+          { !!parent && <PostParentLink parent={ parent }/> }
+          { !parent && meta && <NewThemeSettings currentTheme={meta.theme} onChange={ async value => {
```

Comment:
Currently if you first pick a legacy theme, and then pick a new theme, it's still showing the legacy theme dropdown with the same value afterwards, even though it's not using it.

I'll see if I can improve that. For now you need to keep in mind that if the upper select is not "legacy" but one of the created themes, that one will take precedence.